src/chrome/content/rules/DHL.de.xml

```diff
@@ -1,8 +1,20 @@
 <ruleset name="DHL.de" platform="mixedcontent">
   <target host="dhl.de"/>
-  <target host="*.dhl.de"/>
+  <target host="www.dhl.de"/>
+  <target host="mobile.dhl.de"/>
+  <target host="nolp.dhl.de"/>
+  <target host="nolb.dhl.de"/>
+  <target host="partner.dhl.de"/>
+  <target host="entwickler.dhl.de"/>
+  <target host="fcm.dhl.de"/>
 
   <securecookie host="^(?:.*\.)?dhl\.de$" name=".*" />
 
   <rule from="^http://(?:www\.)?dhl\.de/" to="https://www.dhl.de/"/>
+  <rule from="^http://mobile\.dhl\.de/" to="https://mobile.dhl.de/"/>
```

Comment:
That's correct. Because you no longer include the "*.dhl.de" domain, this would only apply to subdomains listed explicitly in the `target` rules.